components/layout/DocsLayout.js

```diff
@@ -168,11 +168,6 @@ export default function DocsLayout({ post, navItems = {}, children }) {
               <Heading level="h1" typeStyle="heading-lg">
                 {post.title}
               </Heading>
-            {
-              post.isPrerelease 
-              ? <h3 className="text-lxl font-normal text-gray-800 font-sans antialiased">To be released on {post.releaseDate}</h3> 
-              : null
-            }
```

Comment:
I had to remove the support for a release date. Since the date is not anymore in the release branch name, I'm just removing this notice. I don't think it's necessary anyway since the title says it's "Pre-release".

Happy to listen to alternatives.

Reply:
I don't have a problem with that 👌🏼